src/prune/tooling/inspector.cpp

```
@@ -11,32 +11,42 @@
 namespace prune {
 
     void Inspector::draw(
-        GameObjectManager& objects,
+        Scene& scene,
         GridOptions& grid_options,
         const Camera& camera
     ) {
-        draw_selected(objects, grid_options);
+        GameObjectManager& objects = scene.get_object_manager();
+
+        draw_selected(scene, grid_options);
```

Cppcheck (2.20.0)

[style] The function 'available' is never used.

(unusedFunction)